.github/actions/frontend/setup_env/action.yml

```diff
@@ -14,8 +21,17 @@ runs:
         node-version: 18.15.0
         cache: 'npm'
         registry-url: 'https://registry.npmjs.org'
-    - id: npm_install
-      if: ${{ !fromJSON(env.TEST) }}
+    - id: npm_install_prod_deps
```

Comment:
Added per a conversation with @Sivli-Embir . In this case, we opt to default to only installing `prod` dependencies, then the action must specifically choose to install `dev` dependencies via `use_dev`. In this way we can promote the concept of installing only what you truly need for a given action.